### PR TITLE
Parameterized the selinux context: https://github.com/voxpupuli/puppe…

### PR DESCRIPTION
**Pull request by mlosapio**
Addresses issue #51

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -132,6 +132,7 @@
   $https_proxy         = '',
   $priority            = '10',
   $port                = '80',
+  $selinux_context     = 'system_u:object_r:httpd_sys_content_t',
```
Comment (petems) [inline manifests/params.pp]: Lint is giving a `manifests/selinux.pp:25:trailing_whitespace:ERROR:trailing whitespace found`

Can you run puppet-lint -f to fix this or delete the whitespace? 👍 @mlosapio

**Comment by petems** [comment]
@mlosapio Thanks for the fix! Just needs one change 👍

**Comment by mlosapio** [comment]
Commit has been updated.

File: manifests/params.pp (hunk 1 above)
Comment (mlosapio) [inline manifests/params.pp]: done.

**Comment by petems** [comment]
Spectrum is green, Thank you for your contribution! 👍

**Pull request merged**